runtime/js/40_spawn.js

```diff
@@ -21,7 +21,7 @@
 
   const promiseIdSymbol = SymbolFor("Deno.core.internalPromiseId");
 
-  function spawnChild(command, {
+  function spawnChildInner(command, {
```

Comment:
Missing an `apiName` parameter?

Reply:
How so? It's just a helper function that's called by other functions

Reply:
On line 55 within this file, this function (`spawnChildInner`) is called with three arguements; `spawnChild` is a wrapper around `spawnChildInner`, yet `spawnChildInner` doesn't use the API name passed to it?